.eslintrc

```diff
@@ -23,6 +23,8 @@
     "HTMLSelectElement": true,
     "HTMLTextAreaElement": true,
     "Node": true,
+    // Used for uploading profile pictures.
```

Comment:
It's used for this now, but will this be the only thing it's ever used for? Maybe the comment should be more about the fact that it's needed for API requests?